### PR TITLE
09-01-preview changes

### PR DESCRIPTION
**Pull request by hvedati**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by hvedati** [comment]
@VeryEarly Please let me know if this PR is good to merge.

File: src/StackHCI/StackHCIVm/README.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+
+> see https://aka.ms/autorest
+
+```yaml
+require:
+  - E:/testing/readme.azure.noprofile.md
+repo: azure-rest-api-specs-pr
+# lock the commit
+input-file:
+  - E:/azure-rest-api-specs-pr/specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2023-09-01-preview/common.json
```
Comment (VeryEarly) [inline src/StackHCI/StackHCIVm/README.md:62]: do not use local swagger as input

**Review by VeryEarly — CHANGES_REQUESTED** [review]
there is a folder export under docs, please remove this folder and move help markdown under docs

**Review by VeryEarly — CHANGES_REQUESTED** [review]
folder structure is incorrect, there should be folders for both StackHCIVm and StackHCI

File: src/StackHCI/StackHCIVm/README.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+  - E:/azure-rest-api-specs-pr/specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2023-09-01-preview/virtualHardDisks.json
+  - E:/azure-rest-api-specs-pr/specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2023-09-01-preview/virtualMachineInstances.json
+  - E:/azure-rest-api-specs-pr/specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2023-09-01-preview/logicalNetworks.json
+  - E:/azure-rest-api-specs-pr/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-03-15-preview/HybridCompute.json
+
+module-version: 1.1.0
+title: StackHCIVm
+service-name: StackHCIVm
+subject-prefix: $(service-name)
+
```
Comment (VeryEarly) [inline src/StackHCI/StackHCIVm/README.md:76]: add line: root-module-name: $(prefix).StackHCI

**Review by VeryEarly — CHANGES_REQUESTED** [review]
inconsistent term "StackHCIVM" "StackHCIVm"

**Review by VeryEarly — COMMENTED** [review]
why deleting src/StackHCI/examples/Get-AzStackHCILogsDirectory.md?

File: src/StackHCI/StackHCIVm/test/envValues.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+"resourceGroupName": "mkclus0824-rg",
```
Comment (VeryEarly) [inline src/StackHCI/StackHCIVm/test/envValues.json:1]: what is this file for?

**Comment by hvedati** [comment]
> folder structure is incorrect, there should be folders for both StackHCIVm and StackHCI

@VeryEarly  I was told that I need to only include the files for the new StackHCIVm module in the PR. I have had this conversation with other engineers in previous iterations!

**Comment by hvedati** [comment]
> there is a folder export under docs, please remove this folder and move help markdown under docs

I have moved these markdown files just under the doc section.

File: src/StackHCI/StackHCIVm/test/envValues.json (hunk 3 above)
Comment (hvedati) [reply src/StackHCI/StackHCIVm/test/envValues.json]: removed.

File: src/StackHCI/StackHCIVm/README.md [hunk 4]
```diff
@@ -0,0 +1,666 @@
...
+---
+
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+```yaml
+require:
+  - $(this-folder)/../../readme.azure.noprofile.md
+repo: azure-rest-api-specs-pr
```
Comment (VeryEarly) [inline src/StackHCI/StackHCIVm/README.md:59]: use public repo

Comment (hvedati) [reply src/StackHCI/StackHCIVm/README.md:59]: even with the public repo the autogenerated files are the same, they are generated in this format StackHciVM

Comment (hvedati) [reply src/StackHCI/StackHCIVm/README.md:59]: image

**Comment by hvedati** [comment]
PS E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest> .\test-module.ps1 -Playback
Creating isolated process...
Pester v4.10.1
Executing all tests in 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test'

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Add-AzStackHCIVmVirtualMachineDataDisk.Tests.ps1

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Add-AzStackHCIVmVirtualMachineNetworkInterface.Tests.ps1

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmImageNew.Tests.ps1
Running 'playback' mode...
Using recording E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmImageNew.Recording.json

  Describing AzStackHCIVmImageNew
    [+] Create Image  423ms
    [+] List 116ms
    [+] Get 33ms
WARNING: Running this command will delete the image.
    [+] Delete 651ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHciVmLogicalNetwork.Tests.ps1
Running 'playback' mode...
Using recording E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmLogicalNetwork.Recording.json

  Describing AzStackHCIVmLogicalNetwork
    [+] Create Logical Network  143ms
    [+] List 58ms
    [+] Get 54ms
WARNING: Running this command will delete the logical network.
    [+] Delete 317ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHciVmNetworkInterface.Tests.ps1
Running 'playback' mode...
Using recording E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmNetworkInterface.Recording.json

  Describing AzStackHCIVmNetworkInterface
    [+] Create Network Interface   358ms
    [+] List 63ms
    [+] Get 32ms
WARNING: Running this command will delete the network interface.
    [+] Delete 273ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHciVmStoragePath.Tests.ps1
Running 'playback' mode...
Using recording E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmStoragePath.Recording.json

  Describing AzStackHCIVmStoragePath
    [+] Create Storage Path  131ms
    [+] List 51ms
    [+] Get 30ms
WARNING: Running this command will delete the storage path.
    [+] Delete 79ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmVirtualHardDisks.Tests.ps1
Running 'playback' mode...
Using recording E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmVirtualHardDisks.Recording.json

  Describing AzStackHCIVmVirtualHardDisks
    [+] Create 134ms
    [+] List 39ms
    [+] Get 46ms
WARNING: Running this command will delete the virtual hard disk.
    [+] Delete 401ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHciVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Using recording E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\AzStackHCIVmVirtualMachine.Recording.json

  Describing AzStackHCIVmVirtualMachine
    [+] Create Virtual Machine   564ms
    [+] Get 47ms
    [+] Stop 130ms
    [+] Start 96ms
WARNING: Running this command will delete the virtual machine.
    [+] Delete 181ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHCIVmImage.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHCIVmImage.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Get-AzStackHCIVmImage
    [!] BySubscription, is skipped 0ms
    [!] ByName, is skipped 0ms
    [!] ByResourceGroup, is skipped 0ms
    [!] ByResourceId, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMLogicalNetwork.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMLogicalNetwork.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Get-AzStackHciVMLogicalNetwork
    [!] List1, is skipped 0ms
    [!] Get, is skipped 0ms
    [!] List, is skipped 0ms
    [!] ByResourceId, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMNetworkInterface.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMNetworkInterface.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Get-AzStackHciVMNetworkInterface
    [!] List1, is skipped 0ms
    [!] Get, is skipped 0ms
    [!] List, is skipped 0ms
    [!] ByResourceId, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMStoragePath.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMStoragePath.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Get-AzStackHciVMStoragePath
    [!] List1, is skipped 0ms
    [!] Get, is skipped 0ms
    [!] List, is skipped 0ms
    [!] ByResourceId, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMVirtualHardDisk.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHciVMVirtualHardDisk.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Get-AzStackHciVMVirtualHardDisk
    [!] List1, is skipped 0ms
    [!] Get, is skipped 0ms
    [!] List, is skipped 0ms
    [!] ByResourceId, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Get-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Get-AzStackHCIVmVirtualMachine
    [!] BySubscription, is skipped 0ms
    [!] ByName, is skipped 0ms
    [!] ByResourceGroup, is skipped 0ms
    [!] ByResourceId, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmImage.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmImage.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing New-AzStackHCIVmImage
    [!] MarketplaceURN, is skipped 0ms
    [!] GalleryImage, is skipped 0ms
    [!] Marketplace, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmLogicalNetwork.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmLogicalNetwork.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing New-AzStackHCIVmLogicalNetwork
    [!] __AllParameterSets, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmNetworkInterface.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmNetworkInterface.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing New-AzStackHCIVmNetworkInterface
    [!] __AllParameterSets, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmStoragePath.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmStoragePath.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing New-AzStackHCIVmStoragePath
    [!] __AllParameterSets, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmVirtualHardDisk.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmVirtualHardDisk.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing New-AzStackHCIVmVirtualHardDisk
    [!] __AllParameterSets, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\New-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing New-AzStackHCIVmVirtualMachine
    [!] ByImageId, is skipped 0ms
    [!] ByImageName, is skipped 0ms
    [!] ByOsDiskId, is skipped 0ms
    [!] ByOsDiskName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmImage.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmImage.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmImage
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmLogicalNetwork.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmLogicalNetwork.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmLogicalNetwork
    [!] ByResourceId, is skipped 0ms
    [!] Delete, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmNetworkInterface.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmNetworkInterface.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmNetworkInterface
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmStoragePath.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmStoragePath.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmStoragePath
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualHardDisk.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualHardDisk.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmVirtualHardDisk
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmVirtualMachine
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualMachineDataDisk.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualMachineDataDisk.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmVirtualMachineDataDisk
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualMachineNetworkInterface.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Remove-AzStackHCIVmVirtualMachineNetworkInterface.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Remove-AzStackHCIVmVirtualMachineNetworkInterface
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Restart-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Restart-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Restart-AzStackHCIVmVirtualMachine
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Start-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Start-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Start-AzStackHCIVmVirtualMachine
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Stop-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Stop-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Stop-AzStackHCIVmVirtualMachine
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHCIVmImage.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHCIVmImage.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Update-AzStackHCIVmImage
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMLogicalNetwork.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMLogicalNetwork.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Update-AzStackHciVMLogicalNetwork
    [!] ByResourceId, is skipped 0ms
    [!] UpdateExpanded, is skipped 0ms
    [!] UpdateViaIdentityExpanded, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMNetworkInterface.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMNetworkInterface.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Update-AzStackHciVMNetworkInterface
    [!] ByResourceId, is skipped 0ms
    [!] UpdateExpanded, is skipped 0ms
    [!] UpdateViaIdentityExpanded, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMStoragePath.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMStoragePath.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Update-AzStackHciVMStoragePath
    [!] ByResourceId, is skipped 0ms
    [!] UpdateExpanded, is skipped 0ms
    [!] UpdateViaIdentityExpanded, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMVirtualHardDisk.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHciVMVirtualHardDisk.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Update-AzStackHciVMVirtualHardDisk
    [!] ByResourceId, is skipped 0ms
    [!] UpdateExpanded, is skipped 0ms
    [!] UpdateViaIdentityExpanded, is skipped 0ms

Executing script E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHCIVmVirtualMachine.Tests.ps1
Running 'playback' mode...
Recording file 'E:\new\azure-powershell2\src\StackHCI\StackHCIVm.Autorest\test\Update-AzStackHCIVmVirtualMachine.Recording.json' is not present. Tests expecting recorded responses will fail

  Describing Update-AzStackHCIVmVirtualMachine
    [!] ByResourceId, is skipped 0ms
    [!] ByName, is skipped 0ms
Tests completed in 11.53s
Tests Passed: 25, Failed: 0, Skipped: 73, Pending: 0, Inconclusive: 0

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**